careers/careers/wordpress.py

```diff
@@ -0,0 +1,97 @@
+import bleach
```

Comment:
Throw in a comment about this code coming from bedrock so we can make sure we change it there too if we find something wrong here.